extensions/sequelize/README.md

```diff
@@ -128,6 +128,84 @@ contains "Art" in their title, which belongs to category "Programming":
 }
 ```
 
+## SQL Transactions
+
+A Sequelize repository can perform operations in a transaction using the
+`beginTransaction()` method.
+
+### Isolation levels
+
+When you call `beginTransaction()`, you can optionally specify a transaction
+isolation level. It support the following isolation levels:
+
+- `Transaction.ISOLATION_LEVELS.READ_UNCOMMITTED` (default)
+- `Transaction.ISOLATION_LEVELS.READ_COMMITTED`
+- `Transaction.ISOLATION_LEVELS.REPEATABLE_READ`
+- `Transaction.ISOLATION_LEVELS.SERIALIZABLE`
+
+### Options
+
+Following are the supported options:
+
+```ts
+{
+  autocommit?: boolean;
+  isolationLevel?: Transaction.ISOLATION_LEVELS;
+  type?: Transaction.TYPES;
+  deferrable?: string | Deferrable;
+  /**
+   * Parent transaction.
+   */
+  transaction?: Transaction | null;
+}
+```
+
+### Example
+
+```ts
+// Get repository instances. In a typical application, instances are injected
+// via dependency injection using `@repository` decorator.
+const userRepo = await app.getRepository(UserRepository);
+
+// Begin a new transaction.
+// It's also possible to call `userRepo.dataSource.beginTransaction` instead.
+const tx = await userRepo.beginTransaction({
+  isolationLevel: Transaction.ISOLATION_LEVELS.SERIALIZABLE,
+});
+
+try {
+  // Then, we do some calls passing this transaction as an option:
+  const user = await userRepo.create(
+    {
+      firstName: 'Jon',
+      lastName: 'Doe',
+    },
+    {transaction: tx},
+  );
+
+  await userRepo.updateById(
+    user.id,
+    {
+      firstName: 'John',
+    },
+    {transaction: tx},
+  );
+
+  // If the execution reaches this line, no errors were thrown.
+  // We commit the transaction.
+  await tx.commit();
+} catch (error) {
+  // If the execution reaches this line, an error was thrown.
+  // We rollback the transaction.
+  await tx.rollback();
+}
+```
+
+Switching from loopback defaults to sequelize transaction is as simple as
```

Comment:
can we move this example to loopback repo in a separate PR ?

Reply:
Sure. In fact, I will add a more wholesome demo project in the `examples` dir here.

Can we keep this example in readme until then or should I remove it?

Reply:
keep it here but do add it into examples in a separate PR